src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -243,8 +243,8 @@ public long getValidationTimeout()
    @Override
    public void setValidationTimeout(long validationTimeoutMs)
    {
-      if (validationTimeoutMs < 250) {
-         throw new IllegalArgumentException("validationTimeout cannot be less than 250ms");
+      if (validationTimeoutMs < 1000) {
+         throw new IllegalArgumentException("validationTimeout cannot be less than 1000ms");
       }
```

Comment:
for setNetworkTimeout() as in docs, accepts ms but driver should round off to 'nearest' second.
here safe guarding less than 1 second values, faulty driver may round it to ZERO and treat as 'no timeout'